pdns/recursordist/docs/changelog/4.1.rst

```diff
@@ -2,7 +2,7 @@ Changelogs for 4.1.x
 ====================
 
 .. changelog::
-  :version: 4.1.0-alpha2
+  :version: 4.1.0-rc1
```

Comment:
Add some text here about this release e.g. "The RC1 release feature many fixes to the DNSSEC validation code, reported by different users. As well as logging improvements RPZ changes and the Remote Logger."